go/api/connection_management_cluster_commands.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+import "github.com/valkey-io/valkey-glide/go/glide/api/options"
+
+// Supports commands and transactions for the "Connection Management" group of commands for cluster client.
+//
+// See [valkey.io] for details.
+//
+// [valkey.io]: https://valkey.io/commands/#connection
+type ConnectionManagementClusterCommands interface {
+	Ping() (string, error)
+
+	PingWithOptions(pingOptions options.ClusterPingOptions) (ClusterValue[string], error)
```

Comment:
`ping` returns always a single string, even on multi-node route. For reference, see multi-node [response aggregation rules](https://github.com/niharikabhavaraju/valkey-glide/blob/7e2d86d3f7d322f045e77ce21cf04b0cb162a81a/glide-core/redis-rs/redis/src/cluster_routing.rs#L529) and [default routing policy](https://github.com/niharikabhavaraju/valkey-glide/blob/7e2d86d3f7d322f045e77ce21cf04b0cb162a81a/glide-core/redis-rs/redis/src/cluster_routing.rs#L595) in `cluster_routing.rs`.

```suggestion
	PingWithOptions(pingOptions options.ClusterPingOptions) (string, error)
```

Reply:
done